src/v2/view-builder/internals/FormInputFactory.js

```diff
@@ -48,9 +52,29 @@ const createAuthenticatorVerifySelectView = (opt) => {
   };
 };
 
+const createAdminScopesView = () => {
+  return {
+    View: AdminScopeList,
+  };
+};
+const createEnduserScopesView = () => {
+  return {
+    View: View.extend({
+      children () {
+        return [
+          EnduserScopeList,
+          ConsentAgreementText,
+        ];
+      }
+    })
+  };
+};
+
 const inputCreationStrategy = {
   authenticatorEnrollSelect: createAuthenticatorEnrollSelectView,
-  authenticatorVerifySelect: createAuthenticatorVerifySelectView
+  authenticatorVerifySelect: createAuthenticatorVerifySelectView,
+  [RemediationForms.CONSENT_ADMIN]: createAdminScopesView,
```

Comment:
nit:
I'd rather use different naming (InputTypeConstant.consent_admin or just hard-code `consent-admin`) even thought the value is same to avoid confusing.